release_dashboard/lib/widgets/clean_release.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../services/conductor.dart';
+import 'package:flutter/material.dart';
+
+import 'common/dialog_prompt.dart';
+import 'common/snackbar_prompt.dart';
+
+/// Button to clean the current release.
+///
+/// When the button is clicked, a dialogue prompt will open for confirmation.
+/// Clicking on [Yes] will clean the release, [No] will close the dialogue prompt.
+class CleanRelease extends StatefulWidget {
+  const CleanRelease({
+    Key? key,
+    required this.conductor,
+  }) : super(key: key);
+
+  final ConductorService conductor;
+
+  @override
+  State<CleanRelease> createState() => _CleanReleaseState();
+}
+
+class _CleanReleaseState extends State<CleanRelease> {
+  String? _errorMsg;
+
+  /// Updates [_errorMsg] with [errorMsg].
+  void _updateErrorMsg(String? errorMsg) {
+    setState(() {
+      _errorMsg = errorMsg;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.fromLTRB(0, 0, 40, 0),
+      child: IconButton(
+        key: const Key('conductorClean'),
+        icon: const Icon(Icons.delete),
+        onPressed: () {
+          dialogPrompt(
+            context: context,
+            title: 'Are you sure you want to clean up the current release?',
+            content: 'This will abort and delete a work in progress release. This process is not revertible!',
+            leftOptionTitle: 'Yes',
+            rightOptionTitle: 'No',
+            leftOptionCallback: () {
+              _updateErrorMsg('This is a temporary error placeholder');
```

Comment:
Temporary holder to display an error, since there is no other method that calls `_updateErrorMsg`, the error would never displayed otherwise. In the future, the method that calls `cleanContext` will be wrapped in a `try catch` block, and `_updateErrorMsg` will be called in the `catch` block.

Reply:
It's preferable to say "Feature is not implemented. Please use `conductor clean`

The format being a quick description, and an action the user can take to fix it